pages/status.tsx

```diff
@@ -140,7 +140,22 @@ const Status: FC = () => {
                 goBackStyle="primary"
                 checkAgainText={t('check-again')}
               >
-                <p className=" mb-6 text-2xl">{t('unable-to-find-status')}</p>
+                {/* <p className=" mb-6 text-2xl">{t('unable-to-find-status')}</p> */}
+                <div>
+                  <h3 className="mb-3">{t('cannot-provide-result.title')}</h3>
```

Comment:
This should be an H2 in order to retain logical heading order. Currently we're skipping from H1 straight to H3 so an axe scan flags this. You can style it to appear the same but for screen reader purposes it's better for them to have a logical order.